includes/config/include.php

```diff
@@ -28,7 +28,7 @@
 
 define('TP_VERSION', '3.1.2');
 define("UPGRADE_MIN_DATE", "1727110744");
-define('TP_VERSION_MINOR', '6');
+define('TP_VERSION_MINOR', '7');
```

Comment:
Did you forget to bump `TP_VERSION` to 3.1.3?

Reply:
In fact I have not yet decided.
We can have this decision once we decide to make 3.1.2 released.
I'm open to discuss it 😉